presto-hive/src/test/java/com/facebook/presto/hive/TestHiveRecoverableGroupedExecution.java

```diff
@@ -176,6 +180,107 @@ public void testInsertBucketedTable()
                         "DROP TABLE IF EXISTS test_failure"));
     }
 
+    @Test(timeOut = 60_000)
+    public void testCreateUnbucketedTable()
```

Comment:
nit: something like `testCreateUnbucketedTableWithGroupedExecution` ?  @rschlussel  might have better suggestions 😃 . 

ditto for `testInsertUnbucketedTable`